Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/ApplicationConstants.java

```diff
@@ -117,6 +117,9 @@
     public static final String SESSION_CLASSES = "vip-classes";
     public static final String SEPARATOR_INPUT = "##";
     public static final String SEPARATOR_LIST = "@@";
+    
+    // Default result directory
+    public static final String DEFAULT_RESULT_DIRECTORY = "/vip/Home";
```

Comment:
Could use DataManagerConstants -- ROOT et USERS_HOME

Reply:
Yes I agree. Done in the last commit.